packages/veritone-react-common/src/components/SearchBar/TranscriptModal/index.js

```diff
@@ -8,4 +17,117 @@ const getTranscriptLabel = modalState => {
   };
 };
 
-export { getTranscriptLabel };
+
+export default class TranscriptSearchModal extends React.Component {
```

Comment:
I'd also move the TranscriptSearchModal and the TranscriptSearchForm out of index.js and into separate files.